src/xrp-path-element.ts

```diff
@@ -8,27 +8,20 @@ type PathElement = Payment.PathElement
  * - SeeAlso: https://xrpl.org/paths.html
  */
 export default class XRPPathElement {
-  public account: string | null
+  public account: string | undefined
 
-  public currency: XRPCurrency | null
+  public currency!: XRPCurrency | undefined
 
-  public issuer: string | null
+  public issuer: string | undefined
```

Comment:
Can these never be `null`?

Reply:
That's a great question.  I believe they can never be null because they are never assigned null in our code, and the native TypeScript implementations of the Protobuf classes either return a value or undefined with the getters and setters, as you can see here (sry for the black and white)
<img width="799" alt="Screen Shot 2020-03-09 at 5 08 36 PM" src="https://user-images.githubusercontent.com/24928141/76267678-c9148300-6228-11ea-9c4f-98531bc2a5fb.png">